src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int SUPPLY = 0;
+    private static final int BUY = 1;
+    private static final int ACTION = 0;
+    private static final int QUANTITY = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] sourceStrings;
+        int[] results = new int[2];
+        File fileFrom = new File(fromFileName);
+        sourceStrings = readFile(fileFrom).split(" ");
```

Comment:
What if we add a `comment` column and some cells contain `We need to buy more apples`?